test/e2e-tests.sh

```diff
@@ -23,7 +23,7 @@
 # Calling this script without arguments will create a new cluster in
 # project $PROJECT_ID, run the tests and delete the cluster.
 
-source $(dirname $0)/../scripts/e2e-tests.sh
+source $(dirname "$0")/../scripts/e2e-tests.sh
 
```

Comment:
while you are here...... could you replace all the `$0` with `${BASH_SOURCE[0]}` ?

Reply:
done